packages/components/src/tooltip/test/index.tsx

```diff
@@ -65,36 +65,6 @@ describe( 'Tooltip', () => {
 			expectTooltipToBeHidden();
 		} );
 
-		it( 'should associate the tooltip text with its anchor via the accessible description when visible', async () => {
```

Comment:
Can this be done in a separate PR that provides the rationale behind the test removal?

Reply:
Not without this test failing.

Reply:
If we don't know why it's failing but don't want to deal with it immediately, I'd recommend skipping the test( `it.skip()`) and adding a TODO comment to research in another PR.

Reply:
We know why it's failing. Please read the description as it details (or links to the details of) every fix - in this case it's because tooltips don't add the describedBy attribute anymore.

Reply:
Got it, thanks! It seems like it might be a good idea to consider including some of the Ariakit changelog details in our own changelog for clarity.

Reply:
Yeah, I think in this case we'll need to basically copy and paste the tooltip advisory, e.g. "tooltips don't use describedBy anymore etc etc" - that's the only notable change that will be relevant to consumers afaik

Reply:
Maybe we can also invert the test, to basically prove that if the consumer provides `describedBy` on their own, it works well?

Reply:
Hmm, can you give an example of the test you propose in pseudo-code? I fail to see the value of it at the moment, but maybe I'm missing some details and nuance.

Reply:
Maybe what I suggested doesn't make sense. Just to confirm: `describedBy` is no longer being passed to tooltips, and there's no way for consumers to have it if they want (except manually creating the description element and providing an `aria-describedby` prop manually of course)? If that's true, then there's no need to add a test for that and you can ignore my previous comment.

Reply:
We discussed this on a call and agreed to back compat this `aria-describedby` behavior for this PR, and deal with its removal as a separate task 👍

Reply:
done!